lang/python/python-pycparser/Makefile

```diff
@@ -8,11 +8,11 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=python-pycparser
-PKG_VERSION:=2.20
-PKG_RELEASE:=5
+PKG_VERSION:=2.21
+PKG_RELEASE:=1
```

Comment:
i'd start using $(AUTORELEASE) at some point;
i'm also paranoid about it; 
but i also like the idea of the build-system taking care of this for us;